continuity.api/src/main/java/org/continuity/api/rest/RestApi.java

```diff
@@ -1,622 +1,718 @@
-		public static class Model {
-
-			public static final String ROOT = "/model";
-
-			/** {@value #ROOT}/{id} */
-			public static final RestEndpoint OVERVIEW = RestEndpoint.of(SERVICE_NAME, ROOT, Paths.OVERVIEW, RequestMethod.GET);
-
-			/** {@value #ROOT}/{id} */
-			public static final RestEndpoint REMOVE = RestEndpoint.of(SERVICE_NAME, ROOT, Paths.REMOVE, RequestMethod.DELETE);
-
-			/** {@value #ROOT}/{id}/application */
-			public static final RestEndpoint GET_APPLICATION = RestEndpoint.of(SERVICE_NAME, ROOT, Paths.GET_APPLICATION, RequestMethod.GET);
-
-			/** {@value #ROOT}/{id}/annotation */
-			public static final RestEndpoint GET_ANNOTATION = RestEndpoint.of(SERVICE_NAME, ROOT, Paths.GET_ANNOTATION, RequestMethod.GET);
-
-			/** {@value #ROOT}/{id}/persist */
-			public static final RestEndpoint PERSIST = RestEndpoint.of(SERVICE_NAME, ROOT, Paths.PERSIST, RequestMethod.POST);
-
-			private Model() {
-			}
-
-			public static class Paths {
-
-				public static final String OVERVIEW = "/{id}";
-				public static final String REMOVE = "/{id}";
-				public static final String GET_APPLICATION = "/{id}/application";
-				public static final String GET_ANNOTATION = "/{id}/annotation";
-				public static final String PERSIST = "/{id}/persist";
-
-				private Paths() {
-				}
-			}
-
-		}
-
-	}
-
-	/**
-	 * Generic, abstract REST endpoints. The implementing endpoints can be retrieved via the
-	 * {@link Map#get(Object)} method, e.g., the JMeter {@link Model#OVERVIEW} via
-	 * {@link Generic#WORKLOAD_MODEL_LINK}<code>.get("jmeter")</code>.
-	 *
-	 * @author Henning Schulz
-	 *
-	 */
-	public static class Generic {
-
-		/**
-		 * [workload-model-type]/model/{id}
-		 *
-		 * @see RestApi.Wessbas.Model#OVERVIEW
-		 */
-		public static final Map<String, RestEndpoint> WORKLOAD_MODEL_LINK = new HashMap<>();
-
-		/**
-		 * [workload-model-type]/model/{id}/persist
-		 *
-		 * @see RestApi.Wessbas.Model#PERSIST
-		 */
-		public static final Map<String, RestEndpoint> PERSIST_WORKLOAD_MODEL = new HashMap<>();
-
-		/**
-		 * [load-test-type]/loadtest/{id}
-		 *
-		 * @see RestApi.JMeter.TestPlan#GET
-		 */
-		public static final Map<String, RestEndpoint> GET_LOAD_TEST = new HashMap<>();
-
-		/**
-		 * [load-test-type]/report/{id}
-		 *
-		 * @see RestApi.JMeter.Report#GET
-		 */
-		public static final Map<String, RestEndpoint> GET_LOAD_TEST_REPORT = new HashMap<>();
-
-		/**
-		 * [load-test-type]/report/{id}
-		 *
-		 * @see RestApi.JMeter.Report#DELETE
-		 */
-		public static final Map<String, RestEndpoint> DELETE_LOAD_TEST_REPORT = new HashMap<>();
-
-		static {
-			WORKLOAD_MODEL_LINK.put("wessbas", Wessbas.Model.OVERVIEW);
-
-			PERSIST_WORKLOAD_MODEL.put("wessbas", Wessbas.Model.PERSIST);
-
-			GET_LOAD_TEST.put("jmeter", JMeter.TestPlan.GET);
-
-			GET_LOAD_TEST_REPORT.put("jmeter", JMeter.Report.GET);
-
-			DELETE_LOAD_TEST_REPORT.put("jmeter", JMeter.Report.DELETE);
-		}
-
-		private Generic() {
-		}
-
-	}
-
-}
+package org.continuity.api.rest;

```

Comment:
I think something with Git went wrong. The whole class should not be marked as changed.